<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="7245" started-at="2017-09-02T20:44:46Z" finished-at="2017-09-02T20:44:53Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="7245" started-at="2017-09-02T20:44:46Z" finished-at="2017-09-02T20:44:53Z">
      <class name="keyword.NewToursLogInTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:keyword.NewToursLogInTest@3c756e4d]" name="setUp" is-config="true" duration-ms="7206" started-at="2017-09-02T22:44:46Z" finished-at="2017-09-02T22:44:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="LogInTest()[pri:0, instance:keyword.NewToursLogInTest@3c756e4d]" name="LogInTest" duration-ms="8" started-at="2017-09-02T22:44:53Z" finished-at="2017-09-02T22:44:53Z">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[C:\Users\IBM_ADMIN\Desktop\SeleniumSwapna\WebApplicationTesting\src\propertiesNewTours\NewToursLogIn.properties (The system cannot find the path specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: C:\Users\IBM_ADMIN\Desktop\SeleniumSwapna\WebApplicationTesting\src\propertiesNewTours\NewToursLogIn.properties (The system cannot find the path specified)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at keyword.NewToursLogInTest.LogInTest(NewToursLogInTest.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LogInTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:keyword.NewToursLogInTest@3c756e4d]" name="tearDown" is-config="true" duration-ms="15" started-at="2017-09-02T22:44:53Z" finished-at="2017-09-02T22:44:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- keyword.NewToursLogInTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
